.byebug_history

```diff
@@ -0,0 +1,5 @@
+c
+n
+@game.pieces.delete
+@game.pieces.all.delete
+@game.pieces.all
```

Comment:
I had it added to .gitignore locally I suspect the develop branch is missing it and it hosed me when I rebased.